lua/nvim-treesitter/query_predicates.lua

```diff
@@ -1,5 +1,25 @@
 local query = require "vim.treesitter.query"
 
+local html_script_type_languages = {
+  ["importmap"] = "json",
+  ["module"] = "javascript",
+  ["application/ecmascript"] = "javascript",
+  ["text/ecmascript"] = "javascript",
+}
+
+local non_filetype_match_injection_language_aliases = {
+  ex = "elixir",
+  pl = "perl",
+  sh = "bash",
+  uxn = "uxntal",
+  ts = "typescript",
+}
+
+local function get_language_name_from_fenced_code_block_tag(injection_alias)
```

Comment:
can this be named something else, like `get_parser_from_extension` or such?

Reply:
it's not an extension, though. GFM spec calls it an ["info string"](https://github.github.com/gfm/#info-string)

Reply:
how's that?

Reply:
Agreed, but it can be changed into other names, because usage of this can be, for example, in vue/svelte `lang="..."` attributes, where having this directive will be useful. From the viewpoint of using it in otherplaces, it's easier to follow if the functions related to this directive is named to reflect its usage